xbmc/platform/linux/DBusMessage.h

```diff
@@ -56,13 +56,13 @@ class CDBusMessage
   void AppendArgument(const char **arrayString, unsigned int length);
 
   template<typename TFirst>
-  void AppendArguments(const TFirst first)
+  void AppendArguments(const TFirst& first)
   {
     AppendArgument(first);
     // Recursion end
   }
   template<typename TFirst, typename... TArgs>
-  void AppendArguments(const TFirst first, const TArgs... args)
+  void AppendArguments(const TFirst& first, const TArgs... args)
```

Comment:
I think we also want to pass the variadic part as reference:
```suggestion
  void AppendArguments(const TFirst& first, const TArgs&... args)
```

Reply:
done